config/routes.rb

```diff
@@ -3,4 +3,7 @@
     namespace :v1 do
     end
   end
+
+  get '/organizations/public', to: 'organizations#show'
```

Comment:
Please check if the route is in the right place of the file. Because the resulting route should be `http://localhost:3000/api/v1/organizations/public`
You can check the route generated running this command in terminal `rails routes -g organizations`